src/program/schema.graphql

```diff
@@ -652,6 +652,11 @@ input UpdateModule {
 	Keywords in UpdateModule
 	"""
 	keywords: [String!]
+	"""
+	Objectives for the modules
+	"""
+	objectives: String
```

Comment:
This should not be a String type but rather a list of Strings. Please see `input ModuleFields` as an example of what the data structure of the property is.